lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -2475,6 +2475,20 @@ def test_list_report_downloads(self):
         res_json = json.loads(response.content)
         self.assertEqual(res_json, expected_response)
 
+    def test_delete_report_download(self):
+        def do_nothing():
```

Comment:
You don't need need to change this, but alternatively, you could call this `noop`

```python
def noop():
    pas
```